README.md

```diff
@@ -52,7 +52,7 @@ devtools::install_github("tidyverse/dtplyr")
 
 ## Usage
 
-To use dtplyr, I recommend loading dplyr, dtplyr, and data.table:
+To use dtplyr, I recommend loading data.table, dtplyr, and dplyr:
```

Comment:
I think you accidentally did these in the `.md`?

Reply:
They're in the Rmd, too (I just hadn't rendered the md until the second commit).

Reply:
https://github.com/tidyverse/dtplyr/pull/78/files#diff-5140cd39fa2897e01ffbfb8db0999940R49